ros_ign_interfaces/srv/ControlWorld.srv

```diff
@@ -0,0 +1,11 @@
+bool pause                        
```

Comment:
Do you mind calling this message `WorldControl` to match [ignition::msgs::WorldControl](https://github.com/ignitionrobotics/ign-msgs/blob/ign-msgs7/proto/ignition/msgs/world_control.proto)?